insights/specs/datasources/ssl_certificate.py

```diff
@@ -108,10 +108,16 @@ def nginx_ssl_certificate_files(broker):
     Raises:
         SkipComponent: Raised if "ssl_certificate" directive isn't found
     """
-    conf = broker[NginxConfTree]
-    ssl_certs = conf.find('ssl_certificate')
+    ssl_certs = set()
+    confs = broker[Specs.nginx_conf]
+    for conf in confs:
+        for line in conf.content:
+            if 'ssl_certificate' in line:
```

Comment:
How above changing the `sep` to `ssl_certificate ` (end with a white space), to save the below check in L117?

Reply:
Never mind, your code works as well.